### PR TITLE
Feature/devsu 1403 refactor germline reports page

### PR DESCRIPTION
**Pull request by Anna-log7**
For use with the API branch here: https://github.com/bcgsc/pori_ipr_api/pull/8

- Changed germline boards page to use React instead of AngularJS
- Added custom filter to the exported header

### CODE REVIEW

File: app/views/GermlineView/components/Board/columnDefs.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { formatDate } from '../../../../utils/date';
```
Comment (kttkjl) [inline app/views/GermlineView/components/Board/columnDefs.js:1]: could probably update the path here to the new @

File: app/views/GermlineView/components/Board/components/ApiPaginatedTable/components/CheckboxCell/index.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.checkbox-cell {
+  padding: 0 !important;
```
Comment (kttkjl) [inline app/views/GermlineView/components/Board/components/ApiPaginatedTable/components/CheckboxCell/index.scss:2]: are there absolutely no other way than to use !important?

File: app/views/GermlineView/components/Board/columnDefs.js (hunk 1 above)
Comment (kttkjl) [reply app/views/GermlineView/components/Board/columnDefs.js:1]: oh wait this is waiting on the other PR to be merged in ... guess we can just fix it as we go

Comment (Anna-log7) [reply app/views/GermlineView/components/Board/columnDefs.js:1]: The other PR is just so it doesn't show as a linting error - it still works. I can add it here

File: app/views/GermlineView/components/Board/components/ApiPaginatedTable/components/CheckboxCell/index.scss (hunk 2 above)
Comment (Anna-log7) [reply app/views/GermlineView/components/Board/components/ApiPaginatedTable/components/CheckboxCell/index.scss]: It didn't like it without in testing, but now it seems fine ?_?
Anyways, I removed it

File: app/views/GermlineView/components/Board/index.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+
+import ApiPaginatedTable from './components/ApiPaginatedTable';
+import api from '../../../../services/api';
+import columnDefs from './columnDefs';
+import ParamsContext from './components/ParamsContext';
+
+const Board = (): JSX.Element => {
+  const [reports, setReports] = useState([]);
```
Comment (kttkjl) [inline app/views/GermlineView/components/Board/index.tsx:9]: making a note here that this is typed any, but reports have a way complicated structure right?

**Review by kttkjl — APPROVED** [review]

File: app/views/GermlineView/components/Board/index.tsx (hunk 3 above)
Comment (Anna-log7) [reply app/views/GermlineView/components/Board/index.tsx:9]: It's not too complicated for germline reports so I'll add it in

**Review by kttkjl — APPROVED** [review]

**Pull request merged**